.github/workflows/deploy.yml

```diff
@@ -0,0 +1,70 @@
+name: Deploy
+
+on:
+  push:
+    branches: [master]
```

Comment:
I'm guessing this means all jobs get queues on push to master? don't we need a push to branch prod as well? (I don't understand it well enough)

Reply:
The idea is that we aren't going to be using a separate branch for production. It will entirely be based off master

Reply:
ok so I guess once this is merged there will be a button somewhere to promote? well, will approve it then I trust you know :)